apps/ogrinfo_lib.cpp

```diff
@@ -2173,6 +2178,274 @@ char *GDALVectorInfo(GDALDatasetH hDataset,
+            .store_into(psOptions->bDatasetGetNextFeature)
+            .help(_("Enable random layer reading mode."));
+    }
+
+    argParser->add_argument("-dialect")
+        .metavar("<dialect>")
+        .store_into(psOptions->osDialect)
+        .help(_("SQL dialect."));
+
+    // Only for fuzzing
+    argParser->add_argument("-rc")
+        .hidden()
+        .metavar("<count>")
+        .store_into(psOptions->nRepeatCount)
+        .help(_("Repeat count"));
+
+    argParser->add_argument("-al")
+        .store_into(psOptions->bAllLayers)
+        .help(_("List all layers (used instead of having to give layer names "
+                "as arguments)"));
+
+    {
+        auto &group = argParser->add_mutually_exclusive_group();
+        group.add_argument("-so", "-summary")
+            .store_into(psOptions->bSummaryParser)
+            .help(_("Summary only: list all layers (used instead of having to "
+                    "give layer names as arguments)"));
+
+        group.add_argument("-features")
+            .store_into(psOptions->bFeaturesParser)
+            .help(_("Enable listing of features"));
+    }
+
+    argParser->add_argument("-fields")
+        .choices("YES", "NO")
+        .metavar("YES|NO")
+        .action(
+            [psOptions](const std::string &s) {
+                psOptions->aosOptions.SetNameValue("DISPLAY_FIELDS", s.c_str());
+            })
+        .help(
+            _("If set to NO, the feature dump will not display field values."));
+
+    argParser->add_argument("-geom")
+        .choices("YES", "NO", "SUMMARY", "WKT", "ISO_WKT")
+        .metavar("YES|NO|SUMMARY|WKT|ISO_WKT")
+        .action(
+            [psOptions](const std::string &s) {
+                psOptions->aosOptions.SetNameValue("DISPLAY_GEOMETRY",
+                                                   s.c_str());
+            })
+        .help(_("How to display geometries in feature dump."));
+
+    argParser->add_argument("-oo")
+        .append()
+        .metavar("<NAME=VALUE>")
+        .action(
+            [psOptionsForBinary](const std::string &s)
+            {
+                if (psOptionsForBinary)
+                    psOptionsForBinary->aosOpenOptions.AddString(s.c_str());
+            })
+        .help(_("Dataset open option (format-specific)"));
+
+    argParser->add_argument("-nomd")
+        .flag()
+        .action([psOptions](const std::string &)
+                { psOptions->bShowMetadata = false; })
+        .help(_("Suppress metadata printing"));
+
+    argParser->add_argument("-listmdd")
+        .store_into(psOptions->bListMDD)
+        .help(_("List all metadata domains available for the dataset."));
+
+    argParser->add_argument("-mdd")
+        .append()
+        .metavar("<domain>")
+        .action([psOptions](const std::string &s)
+                { psOptions->aosExtraMDDomains.AddString(s.c_str()); })
+        .help(_("List all metadata domains available for the dataset."));
```

Comment:
help is the same of the previous `-listmdd`  , is that correct?

Reply:
no, changed to "List metadata in the specified domain."